### PR TITLE
fix: Attached to name field 

### PR DESCRIPTION
**Pull request by hitenshKharva**
Ref: [https://app.asana.com/0/1199082161983365/1199376239709583](url)

Edited modification date for job applicant doctype in reference to previous merged [PR](https://github.com/Bloomstack/erpnext/pull/823)

### CODE REVIEW

**Comment by hrwX** [comment]
@hitenshKharva resolve conflicts and change the PR title

**Comment by hrwX** [comment]
- Create a new PR you have pushed to the merged branch, else rebase this branch.
- Resolve merge conflicts

File: erpnext/hr/doctype/job_applicant/job_applicant.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ frappe.ui.form.on("Job Applicant", {
 				}
 			};
 		});
-
+	
+		frm.set_df_property('resume_attachment', 'reqd', frm.is_new() ? 0 : 1)
```
Comment (vishdha) [inline erpnext/hr/doctype/job_applicant/job_applicant.js:46]:
```suggestion
		frm.toggle_reqd('resume_attachment', !frm.is_new());
```

**Review by vishdha — CHANGES_REQUESTED** [review]

**Review by vishdha — CHANGES_REQUESTED** [review]
@hitenshKharva  Remove conflicts

**Review by vishdha — CHANGES_REQUESTED** [review]
@hitenshKharva You need to do the same for webform.

**Comment by neilLasrado** [comment]
@hitenshKharva can you please resolve the merge conflicts?

**Comment by hitenshKharva** [comment]
@neilLasrado @vishdha Closing this branch. New updated PRs for the same have been raised and merged on develop and staging.

**Pull request closed without merging**